### DISCUSSION TITLE
Control 3 led panel parallel ?

### DISCUSSION THREAD

**Discussion by paulpham1407** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#567] [Q&A]
I am looking to control 3 LED panels independently. How would displaying a video in a 192x128 format work with an ESP32?

**Comment by board707** [comment]
Please clarify your request. Why do you need to control panels "in parallel" ? 
Normally led panels connects in series, you don't need a parallel connection to display the video on 3 panels

**Comment by paulpham1407** [comment]
I don't know, so I want to connect 3 boards so that I can run a complete vertical video with 3xPanel using an ESP32.
![Screenshot 2024-01-10 161146](https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/assets/124969647/5ad76d22-0d1d-4725-b227-2b40e8b7dd20)

**Comment by board707** [comment]
So you don't need to control panels independently.
Connect 3 panels in series and you can use it as a whole big screen.

**Comment by paulpham1407** [comment]
How to show one video with 3 chain? Do you have example, plesaes

**Comment by board707** [comment]
No, I don't
Wait for @mrfaptastic , he is maintainer of the library

**Comment by mrcodetastic** [comment]
I've gone out of my way to document how this is done.  Please read the chained panels PDF.
